packages/sil-voting-app/src/routes/dashboard/components/AsideFilter.tsx

```diff
@@ -1,33 +1,41 @@
 import { Drawer, List, ListItem, ListItemText } from '@material-ui/core';
```

Comment:
Here I can see the AsideFilter, where is the "header" I assume you have coded it in a separate PageLayout (or something like that), right?

Reply:
The header is in `src/components/Header.tsx` as you suggested in a frontend weekly call.